src/webgl/p5.DataVector.js

```diff
@@ -0,0 +1,94 @@
+import p5 from '../core/main';
+
+/**
+ * An internal class to store data that will be sent to a p5.RenderBuffer.
+ * Those need to eventually go into a Float32Array, so this class provides a
+ * variable-length array container backed by a Float32Array so that it can be
+ * sent to the GPU without allocating a new array each frame.
+ *
+ * Like a C++ vector, its fixed-length Float32Array backing its contents will
+ * double in size when it goes over its capacity.
+ */
```

Comment:
Does this class have use outside of p5 rendering internals? Worth exposing more broadly or at least making an example for future developers here?

Reply:
I think it's probably only useful for things that need a Flaot32Array, which for now is just rendering. An example for devs can't hurt!